ui/src/message_popup/author/new_question_page.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import QuestionPage from './question_page.jsx';
+import {allArchivedQuestions} from './archived_questions.js';
+import {allQuestions} from '../questions.js';
+import * as Api from '../../helpers/api.js';
+
+
+export default React.createClass({
+  displayName: 'NewQuestionPage',
+
+  getInitialState(){
+    return ({
+      loaded: false,
+      allDatabaseQuestions: {
```

Comment:
I'm not following why `allQuestions` and `allArchivedQuestions` are here in `getInitialState`, can this be removed now that this loads them from the server?